include/qpdf/QPDFObjectHandle.hh

```diff
@@ -163,8 +163,8 @@ class QPDFObjectHandle
     // writeToken(). Example:
     // writeToken(QPDFTokenizer::Token(QPDFTokenizer::tt_name, "/text/plain"))
     // would write `/text#2fplain`, and
-    // writeToken(QPDFTokenizer::Token(QPDFTokenizer::tt_string, "a\\(b"))
-    // would write `(a\(b)`
+    // writeToken(QPDFTokenizer::Token(QPDFTokenizer::tt_string, "a\\(b)")
```

Comment:
actually, this one is correct as written -- the ) belongs outside. This is an example of creating a string with an unbalanced parentheses.

Reply:
Should have been obvious if I had thought about it.